### DISCUSSION TITLE
QSO Duplicate Matching issues for POTA operator

### DISCUSSION THREAD

**Discussion by T012m3n7oR** [magicbug/Cloudlog#1736] [Ideas / Feature Requests]
Hello All,

Just wondering if anyone else ran into this, the duplicate checker when importing an ADIF file is removing some of the QSOs in the file.  The Callsign, Date, Band, Freq all match, the difference is the SIG/SIG_INFO fields are different to denote that the operator was in multiple parks at the same time.

Is this something that is a worthwhile change?  Yes, I could have disabled the duplicate checker as well, but I wasn't aware I needed to.

Thanks,
Pat

---
This could be changed in [Logbook_model](https://github.com/magicbug/Cloudlog/blob/13f76d23764bf1ba8d4f3a63b9f4868086d7111e/application/models/Logbook_model.php#L2503)

**Comment by phl0** [comment]
So you are logging the QSO multiple times with exact same data except SIG and SIG_INFO?

**Reply by T012m3n7oR** [reply]
Yes, when logging contacts to an operator that is in multiple valid locations, the only difference is typically the SIG/SIG_INFO for the park reference.

**Reply by T012m3n7oR** [reply]
Here is the information I could find on it.  https://docs.pota.app/docs/activator_reference/park_2_park.html#matching-rules

Previously they had a multiple reference guide that explained to submit your logs while in multiple references, just edit the adif and change the MY_SIG to the second reference -> So to log that as a hunter, I would need multiple records, with different SIG values.

**Comment by AndreasK79** [comment]
To be sure that it is no duplicate at all, all fields (except id) would need to be checked. On the other hand, how many of the fields would realistically be needed? Since this is an issue with POTA, I guess SIG/SIG_INFO could be added to this check. SAT it not check either.
